src/components/modals/TradePreviewModalGP.vue

```diff
@@ -1,70 +1,189 @@
+                @click="showSummaryInFiat = false"
+              >
+                {{ $t('tokens') }}
+              </div>
+              <div
+                :class="showSummaryInFiat && 'text-blue-600 font-medium'"
+                class="pl-1 cursor-pointer"
+                @click="showSummaryInFiat = true"
+              >
+                {{ FiatCurrency.USD }}
+              </div>
+            </div>
+          </div>
+        </template>
+        <div class="p-3 text-sm">
+          <div class="summary-item-row">
+            <div>{{ $t('amountBeforeFees') }}</div>
+            <div>
+              {{ summary.amountBeforeFees }}
+            </div>
+          </div>
+          <div class="summary-item-row">
+            <div>{{ $t('gasCosts') }}</div>
+            <div v-if="trading.isGnosisTrade.value" class="text-green-400">
+              {{ fNum('0', showSummaryInFiat ? 'usd' : 'token') }}
+            </div>
+            <div v-else>
+              <!-- TODO: in order to calculate this I need to get gasLimit of the transaction + gwei price -->
+              Calculated on confirmation
+            </div>
+          </div>
+          <div class="summary-item-row">
+            <div>{{ $t('solverFees') }}</div>
+            <div>
+              {{ summary.solverFees }}
+            </div>
+          </div>
         </div>
-        <div class="mt-6 mb-3 text-sm">
-          Requires {{ requiresApproval ? 2 : 1 }}
-          {{ requiresApproval ? 'transactions' : 'transaction' }}:
+        <template v-slot:footer>
+          <div class="w-full p-3 rounded-b-lg bg-white text-sm">
+            <div class="summary-item-row font-medium">
+              <div>
+                {{
+                  trading.exactIn.value
+                    ? $t('expectedReceiveNoSlippage')
+                    : $t('expectedSellNoSlippage')
+                }}
+              </div>
+              <div>
+                {{ summary.totalWithoutSlippage }}
+              </div>
+            </div>
+            <div class="summary-item-row">
+              <div>
+                {{
+                  trading.exactIn.value
+                    ? $t('minReceivedWithSlippage', [slippageRatePercent])
+                    : $t('maxSoldWithSlippage', [slippageRatePercent])
+                }}
+              </div>
+              <div>
+                {{ summary.totalWithSlippage }}
+              </div>
+            </div>
+          </div>
+        </template>
+      </BalCard>
+      <BalCard shadow="none">
+        <div class="font-semibold mb-2">
+          Requires {{ trading.requiresApproval.value ? 2 : 1 }}
+          {{ trading.requiresApproval.value ? 'transactions' : 'transaction' }}:
         </div>
-        <div>
+        <div class="text-sm">
           <div
-            v-if="requiresApproval"
-            class="p-3 flex items-center border rounded-lg"
+            v-if="trading.requiresApproval.value"
+            class="flex items-center mb-2"
```

Comment:
I think the requires transactions section should be the same style as the summary, no colon and a line break.
<img width="434" alt="Screenshot 2021-07-05 at 19 24 07" src="https://user-images.githubusercontent.com/2406506/124507919-baab8c80-ddc6-11eb-994a-f619fa23b32c.png">